fileServer.js

```diff
@@ -0,0 +1,113 @@
+ 'use strict';
+
+ var http = require('http');
+ var fs = require('fs');
+ var path = require('path');
+ var url = require('url');
+ var rootDir = __dirname + '/public';
+
+ const mimeType = {
+   'html': 'text/html',
+   'jpeg': 'image/jpeg',
+   'jpg': 'image/jpeg',
+   'png': 'image/png',
+   'js': 'text/javascript',
+   'css': 'text/css'
+ };
+
+ function send404Request(res) {
+   fs.readFile('public/404.html', 'UTF-8', function(err, html) {
+<<<<<<< HEAD
```

Comment:
Looks like you have conflict markers checked in with your code! These need to be removed.